docs/metastore_upgrade_guide.md

```diff
@@ -0,0 +1,213 @@
+## Table Upgrade Guide
+This document provides a guide for upgrading the Hive metastore objects to UC using UCX.
+The metastore upgrade process is composed of multiple steps.
+To effectively upgrade the metastores four principal operations are required:
+1. Assess - In this step, you will evaluate the existing HMS tables identified for upgrade so that we can determine the right approach for upgrade. This step is a prerequisite and is performed by the assessment workflow.
```

Comment:
Future or present? You or we/ucx?

```suggestion
1. Assess - In this step, you evaluate the existing HMS tables identified for upgrade to determine the right approach for upgrade. This step is a prerequisite and is performed by the assessment workflow.
```

```suggestion
1. Assess - In this step, you will evaluate the existing HMS tables identified for upgrade so that UCX can determine the right approach for upgrade. This step is a prerequisite and will be performed by the assessment workflow.
```